grails-app/views/layouts/generic.gsp

```diff
@@ -4,19 +4,19 @@
         <meta charset="utf-8">
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-        <g:render template="/manifest" plugin="elurikkus-commons" />
+        <g:render template="/manifest" plugin="elurikkus-commons"/>
```

Comment:
I couldn't find a way to tell my auto formatter otherwise. But I'm going to stop using it anyway because it doesn't get along with grails templates